javascript-custom-methods/calculator.test.js

```diff
@@ -0,0 +1,141 @@
+/* global calculator, expect */
+
+var calculator = {
+  add: function(x, y) {
+    return x + y;
+  },
+
+  subtract: function(x, y) {
+    return x - y;
+  },
+
+  multiply: function(x, y) {
+    return x * y;
+  },
+
+  divide: function(x, y) {
+    return x / y;
+  },
+
+  square: function(x) {
+    return x * x;
+  },
+
+  sumAll: function(numbers) {
+    var sum = 0
+    for (var i = 0; i < numbers.length; i++) {
+      sum += numbers[i]
+    }
+    return sum
+  },
+
+  getAverage: function(numbers) {
+    var average = 0
+    for (var i = 0; i < numbers.length; i++) {
+      average += numbers[i]
+    }
+    return average / numbers.length;
+  }
+}
```

Comment:
This should not be in the `.test.js` file, it should be in the normal js file named `calculator.js`